src/RandNumGenerator.java

```diff
@@ -10,27 +10,72 @@ public static void main (String[] args) {
 
         //welcome message
         System.out.println("Welcome to Ruth's Simple Random Number Generator!");
+        System.out.println("Note, your bounds will be included in the range");
 
         //getting lower bound of range
-        System.out.println("Please input the lower bound of your range (i.e. 3)");
-        int lowerNum = mrScanner.nextInt();
-        System.out.println("The lower bound is: " + lowerNum);
-
-        //getting upper bound of range
-        System.out.println("Please input the upper bound of your range (i.e. 10)");
-        int upperNum = mrScanner.nextInt();
-        System.out.println("The upper bound is: " + lowerNum);
-
-        //getting amount of numbers
-        System.out.println("How many random numbers would you like?");
-        int amtNum = mrScanner.nextInt();
-        System.out.println("Generating " + amtNum + " number(s):");
-
-        int i = amtNum;
-        while (i > 0) {
-            int randNum = lowerNum + rand.nextInt(upperNum - lowerNum);
-            System.out.println(randNum);
-            i--;
+
+        int neg = 1;
```

Comment:
I see you are using this variable as a `yes/no flag.`
Have you thought about `boolean`? The best practice is not to "waste" your CPU resource. When we use `int` the resource will be allocate a lot more, while `boolean` only need 2 states `true/false`